infrastructure/aws/lambda/Dockerfile

```diff
@@ -30,8 +30,11 @@ RUN rm -rdf /asset/numpy/doc/ /asset/bin /asset/geos_license /asset/Misc
 RUN rm -rdf /asset/boto3*
 RUN rm -rdf /asset/botocore*
 
-# Remove system dependencies
-RUN yum remove -y gcc-c++
+# Strip debug symbols from compiled C/C++ code (except for numpy.libs!)
+RUN cd /asset && \
+  find . -type f -name '*.so*' \
+  -not -path "./numpy.libs/*" \
+  -exec strip --strip-unneeded {} \;
```

Comment:
I made this change in `titiler` dockerfile and was able to successfully import `rasterio`, `numpy`, `zarr`, and `xarray` without any issues. It did NOT work if I ran `strip` on the `numpy.libs` directory, though.